carbon-design-system/carbon
# docs(Slug): move slug examples under experimental section

Based on feedback from @mattrosno. This moves all slug-specific stories under the `unstable__slug` storybook folder. It also adds comments to components with a `slug` prop to mark it as an experimental prop. Also closes https://github.com/carbon-design-system/carbon/issues/15296

#### Changelog

**New**

- A new `Examples` folder under `unstable__Slug` in storybook. This contains an example of each component that has the `slug` prop. 

**Changed**

- Moved the `AI Form` story to under `unstable__Slug`, as well as all associated styles
- Moved the `Tile` slug test story to under `unstable__Slug`, as well as all associated styles
- Moved the `Radio Button` and `Checkbox` slug test stories to under `unstable__Slug`, as well as all associated styles
- Added `Experimental: ` before each `slug` prop type definition

**Removed**

- Removed an arg type definition in `Checkbox` story file that was causing the `disabled` prop to be hidden (#15296)
- All slug test/example stories that were located under components other than `Slug`

#### Testing / Reviewing

- Ensure the existing Tile and Form stories do not have any regressions with the removal of the story-style files. 
- Ensure the components render as expected inside `unstable__Slug --> Form`
- Ensure the components render as expected inside `unstable__Slug --> Examples` and the props work as expected
